tests/bin/route53-wildcard.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""
+The route53-wildcard utility is used to add and remove wildcard
+DNS entries from an AWS Route53 zonefile.
+
+Usage: route53-wildcard.py <action> <name> <value> [options]
+
+Options:
+
+  --zone=<zone>      name of zone to use [defaults to parsing name]
+  --region=<region>  AWS region to use [default: us-west-2].
+  --ttl=<ttl>        record TTL to use [default: 60].
+
+Examples:
+
+./route53-wildcard.py create mycluster.gabrtv.io deis-elb.blah.amazonaws.com
+./route53-wildcard.py delete mycluster.gabrtv.io deis-elb.blah.amazonaws.com
+"""
+
+from __future__ import print_function
+
+import boto.route53
+import docopt
+import socket
+import time
+import uuid
+
+
+def parse_args():
+    return docopt.docopt(__doc__)
+
+
+def create_cname(args):
+    conn = boto.route53.connect_to_region(args['--region'])
+    zone = conn.get_zone(args['<zone>'])
+    name = args['<name>']
+    status = zone.add_cname(name, args['<value>'], ttl=args['--ttl'])
+    print("waiting for record to sync: {}".format(status))
+    while status.update() != "INSYNC":
+        time.sleep(2)
+    print(status)
+    print('waiting for wildcard domain to become available...', end='')
+    # AWS docs say it can take up to 30 minutes for route53 changes to happen, although
+    # it seems to be almost immediate.
+    for i in xrange(120):
```

Comment:
This will wait a long time as AWS docs suggest, but in practice returns on the first loop since the domain is available after the zone's status goes to `INSYNC`. If the route53 domain somehow isn't actually available, the script that calls this code will print some type of socket error, so I'm not sure that's needed.